build.gradle

```diff
@@ -2,45 +2,42 @@ group 'com.SlackUserPresenceManager'
 version '1.2-SNAPSHOT'
 
 apply plugin: 'java'
-apply plugin: 'idea'
+apply plugin: 'kotlin'
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
-
-repositories {
-    mavenCentral()
-}
-
-idea {
-    module {
-        downloadJavadoc = true
-        downloadSources = true
+buildscript {
+    ext.kotlin_version = '1.2.0'
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
-}
-
-dependencies {
-    compile group: 'org.apache.httpcomponents', name: 'fluent-hc', version: '4.5.3'
-    compile group: 'com.google.code.gson', name: 'gson', version: '1.7.2'
-    compile group: 'net.java.dev.jna', name: 'jna-platform', version: '4.4.0'
-    compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.8.2'
 }
 
 jar {
     manifest {
-        attributes "Main-Class": "com.slackuserpresencemanager.Main"
+        attributes "Main-Class": "com.slackuserpresencemanager.MainKt"
     }
 
     from {
         configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
     }
+}
 
-    copy {
-        from "SlackUserPresenceManager.properties"
-        into "build/libs"
-    }
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+compileKotlin.kotlinOptions.jvmTarget = "1.8"
+compileTestKotlin.kotlinOptions.jvmTarget = "1.8"
+
+repositories {

```

Comment:
Just wanted to test a comment in a gh PR

# Testing a header

#6 
Test reference another issue.

Reply:
Starting a refview